indexer/indexer.gen.go

```diff
@@ -2647,14 +2708,15 @@ func (c *indexerGatewayClient) GetTokenBalances(ctx context.Context, chainIds []
 	return out.Ret0, out.Ret1, err
 }
 
-func (c *indexerGatewayClient) GetTokenBalancesSummary(ctx context.Context, chainIds []uint64, networks []string, filter *TokenBalancesFilter, omitMetadata *bool, page *Page) (*Page, []*GatewayNativeTokenBalances, []*GatewayTokenBalance, error) {
+func (c *indexerGatewayClient) GetTokenBalancesSummary(ctx context.Context, chainIds []uint64, networks []string, networkType *NetworkType, filter *TokenBalancesFilter, omitMetadata *bool, page *Page) (*Page, []*GatewayNativeTokenBalances, []*GatewayTokenBalance, error) {
```

Comment:
This is an expected breaking change in the client. Discussed at https://sequence-xyz.slack.com/archives/C06LAL1UMLJ/p1744815286782789